hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
```

Comment:
yes , but in another PR ( I guess ) you told me that the indentation is wrong ( where it was 2 spaces ) so I thought you have different configuration for js files and you use 4 spaces instead of 2 as for php files  but that configuration is not specified in the editoconfig file .